### PR TITLE
20 eslint not working in the v04 branch

### PR DESCRIPTION
**Pull request by layaxx**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -48,13 +50,8 @@ module.exports = {
     {
       files: ['admin/src/**/*.js'],
       parser: '@babel/eslint-parser',
-      extends: [
-        'eslint:recommended',
-        'plugin:react/recommended',
-        'plugin:redux-saga/recommended',
```
Comment (layaxx) [inline .eslintrc.js:54]: we dont use redux or redux-saga enywhere, so this should not be needed

File: .eslintrc.js [hunk 2]
```diff
@@ -24,6 +24,12 @@ module.exports = {
       },
       rules: {
         'node/no-unpublished-require': 'off',
+        'node/no-missing-require': [
+          'error',
+          {
+            allowModules: ['@strapi/strapi'],
```
Comment (layaxx) [inline .eslintrc.js:30]: this should be fine since `@strapi/strapi` is a peerDependency

**Comment by layaxx** [comment]
added Pipeline for ESLint

File: .eslintrc.js (hunk 2 above)
Comment (sargreal) [reply .eslintrc.js:30]: Looks good

File: server/utils/__tests__/update-uids.test.js [hunk 3]
```diff
@@ -31,6 +18,7 @@ const defaultUid = 'unique-uid'
 const contentManagerPluginMock = {
   services: {
     uid: () => ({
+      // eslint-disable-next-line no-unused-vars
       generateUIDField({ contentTypeUID, field, data }) {
```
Comment (sargreal) [inline server/utils/__tests__/update-uids.test.js:22]: I believe we can remove the unused vars here right?
```suggestion
      generateUIDField() {
```

File: .eslintrc.js (hunk 1 above)
Comment (sargreal) [reply .eslintrc.js:54]: Yes makes sense

**Review by sargreal — CHANGES_REQUESTED** [review]

**Review by sargreal — APPROVED** [review]

**Pull request merged**